File: server/mails/welcomeMail.js
const nodemailer = require('nodemailer');

// Function to send email
const sendWelcomeEmail = async (toEmail) => {
  try {
    // Create a transporter object using SMTP transport
    let transporter = nodemailer.createTransport({
      host: 'smtp.gmail.com',
      port: 587,
      secure: false, // true for 465, false for other ports
      auth: {
        user: 'shvwxyz05@gmail.com', // Your email
        pass: 'jqpr nran qhfk ohrs'  // Your email password or app-specific password
      }
    });

    // Email options
    let mailOptions = {
      from: '"Thought Bank" <shvwxyz05@gmail.com>', // sender address
      to: 'gshvamsi@gmail.com' , // list of receivers
      subject: 'Welcome to Thought Bank', // Subject line
      html: `
        <html>
          <body>
            <p>Welcome to Thought Bank!<br>
               Please visit our website by clicking on the link below:<br>
               <a href="http://www.thoughtbank.com">Thought Bank</a>
            </p>
          </body>
        </html>
      ` // HTML body
    };

    // Send email
    let info = await transporter.sendMail(mailOptions);

    console.log('Message sent: %s', info.messageId);
  } catch (error) {
    console.error('Error sending email:', error);
  }
};

module.exports = sendWelcomeEmail;

File: server/python/welcomeMail.py
import smtplib
from email.mime.multipart import MIMEMultipart
from email.mime.text import MIMEText

smtp_object = smtplib.SMTP('smtp.gmail.com', 587)
smtp_object.ehlo()
smtp_object.starttls()

fromemail = "shvwxyz05@gmail.com"
toemail = "gshvamsi@gmail.com"
password = "jqpr nran qhfk ohrs"  

smtp_object.login(fromemail, password)

sub = "Thought Bank"
message = "Welcome to Thought Bank"

msg = MIMEMultipart()
msg['From'] = fromemail
msg['To'] = toemail
msg['Subject'] = sub
msg.attach(MIMEText(message, 'plain'))  

smtp_object.sendmail(fromemail, toemail, msg.as_string())

smtp_object.quit()